draft-ietf-intarea-provisioning-domains.xml

```diff
@@ -962,6 +954,25 @@
 		  require Standards Action <xref target="RFC8126">RFC8126</xref>, via
 		  a Standards Track RFC document.</t>
 	  </section>
+
+	  <section title="PvD JSON Media Type Registration">
+		  <t>This document registers the media type for PvD JSON text, "application/pvd+json".</t>
+		  <t>Type Name:  application</t>
+		  <t>Subtype Name:  pvd+json</t>
+		  <t>Required parameters:  None</t>
+		  <t>Optional parameters:  None</t>
+		  <t>Encoding considerations:  Encoding considerations are identical to those specified for the "application/json" media type.</t>
+		  <t>Security considerations:  See <xref target="security"/>.</t>
+		  <t>Interoperability considerations:  This document specifies format of conforming messages and the interpretation thereof.</t>
+		  <t>Published specification:  This document</t>
+		  <t>Applications that use this media type:  This media type is intended to be used by network advertising additional Provisioning Domain information, and clients looking up such information.</t>
+		  <t>Additional information:  None</t>
+		  <t>Person and email address to contact for further information:  See Authors' Addresses section</t>
+		  <t>Intended usage:  COMMON</t>
+		  <t>Restrictions on usage:  None</t>
+		  <t>Author:  IETF</t>
+		  <t>Change controller:  IETF</t>
+	  </section>
```

Comment:
I wonder if using a different media type could cause issues in existing parsers.

Reply:
I think this should be pretty safe; adding a media type seems to be pretty standard practice.